package.json

```diff
@@ -38,7 +37,7 @@
     "good": "^5.1.2",
     "good-console": "^4.1.0",
     "handlebars": "^3.0.0",
-    "hapi": "^8.2.0",
+    "hapi": "8.6.1",
```

Comment:
at one point, hapi@8.8.x was installed and shot (used for request injection during testing) broke. locking this down to prevent bad installs